client/app/components/Navbar/index.tsx

```diff
@@ -70,8 +72,10 @@ const Navbar = () => {
                     >Services</a></li>
                     <li className={style.navItem}><a onMouseLeave={handleMouseLeave} onMouseEnter={() => handleMouseEnter(INDUSTRY_JSON_PATH)}
                     >Industries</a></li>
-                    <li className={style.navItem}><a onMouseLeave={handleMouseLeave} onMouseEnter={() => handleMouseEnter(ABOUT_JSON_PATH)}
-                    >About Us</a></li>
+                    <li className={style.navItem}><Link href="/AboutUs" onMouseLeave={handleMouseLeave}>
```

Comment:
cred ca poti s lasi dropdown-ul pe about us, probabil va trebui sa facem pagini pentru fiecare chestie de acolo

adica noi in about us dropdown avem about blog si contact, in about din dropdown si pe about us mare ar trebui sa fie link-ul pe on click